nested-passthrough/ansible/setup_gpu/tasks/main.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
AFAICT, this file is unused? Is this so you can use setup_gpu as a stand-alone role? If so, why are you preferring to do `tasks_from` on each file (in setup_host.yml) rather than just including the top-level role and relying on this file to include your tasks?

Also noticed that this file leaves out iommu.yml, which is mentioned in setup_host.yml without an obvious reason (to me).

I feel like I might be missing something here, ansible plays tricks on me sometimes. :)

Reply:
In practice it doesn't really matter. You could just call main, unless you have a good reason to just use some subset of tasks implemented in the role. In this case we really do want to call all of them.

But I do assume that both playbook and role will get more complex, as we support more hardware combos. At that point it may be necessary to work with conditionals. 

In our ansible collections those end up in playbook, while the main.yml remains as as sort *base* case. 
I'm trying to follow that pattern here, reasoning that it may make things easier in the future.

As for the iommu.yml, yeah I forgot about that one.